### DISCUSSION TITLE
Underscore in domain name

### DISCUSSION THREAD

**Discussion by firasm** [plausible/analytics#2122] [Anything Else]
Hello all,

It seems I'm unable to add underscores (`_`) to my domain name when adding a site on Plausible. Could someone clarify if this field is just the "title" of my site, or if the actual domain name entered there matters.

In other words, can I just enter something like 'mySiteName' if my actual site name is 'site.com/my_site_name' ?

**update**: actually I just tried it and it seems to work just fine...
<img width="388" alt="Screen Shot 2022-08-18 at 12 08 57 PM" src="https://user-images.githubusercontent.com/2507459/185475042-efe36cc4-a2a1-4df8-a139-64bb32d61d82.png">

I wonder if this will solve all the domain renaming issues people are having (specifically because the field doesn't fit in the card dashboard so it's hard to tell which site is which) - perhaps this is because when you first try to create a site, it implies that the domain should be a real website, but really it's just a "title"

<img width="541" alt="Screen Shot 2022-08-18 at 12 10 10 PM" src="https://user-images.githubusercontent.com/2507459/185475178-1c42b131-af2c-43ab-bc15-f1cbb520e533.png">

**Update:** Umlauts and other special characters (including numbers, dots and slashes) are now supported so you can add international domain names to Plausible too.

**Comment by metmarkosaric** [comment]
hi @firasm! the way you did it would work all fine. that field is more like a site ID in our database and doesn't necessary need to reflect the exact URL of your site

**Reply by firasm** [reply]
Sounds good - thanks for confirming! If you want, I can make a PR with a couple of sentences about this in the docs since it's somewhat deceptive (probably unintentionally)

**Reply by metmarkosaric** [reply]
no problem! the regular approach of adding the site domain exactly as it is works best but there's a note about special cases already on [this page](https://plausible.io/docs/add-website) for international characters etc that we don't support yet

**Comment by metmarkosaric** [comment]
**Update:** Umlauts and other special characters (including numbers, dots and slashes) are now supported so you can add international domain names to Plausible too.

**Reply by firasm** [reply]
Thank you!
